Makefile

```diff
@@ -1,13 +1,41 @@
-PWD         := $(shell pwd)
+MHOME         := $(shell pwd)
```

Comment:
This isn't used, and is also a duplicate of PWD.

Reply:
Sorry, I forgot to remove it. It was for testing.